### ISSUE TITLE
Add a header to select filters

### ISSUE THREAD

**Issue by Pavel401** [Pavel401/Jobs-Scraper#18]
Create a header to select Filters (Company, Location,..)

**Comment by hemant933** [comment]
assign it to me sir

**Comment by Tholkappiar** [comment]
Hi @assignee_username,

I hope this message finds you well. I wanted to let you know that I've completed the work on this issue/pull request. The changes have been implemented, and I've thoroughly tested them.

If the Assigned Persons good with this , let me know please !

Thank you for your time, and I look forward to your feedback!

Best regards,
Tholkappiar

[Screencast from 15-01-24 01:22:26 PM IST.webm](https://github.com/Pavel401/Jobs-Scraper/assets/114584730/8323e74c-dfc7-4338-9758-6fc928d86435)

**Comment by Pavel401** [comment]
@hemant933 Shall I assign this issue to @Tholkappiar as he has completed the work?

**Comment by Tholkappiar** [comment]
> @Tholkappiar amazing work 🚀

#### Can i take this issue , i have completed my work ! it would be nice if you'r okay with this ! Thank you , waiting for your response !

**Comment by Tholkappiar** [comment]
> ofcourse i don't have any problem

#### Thank you very much @hemant933 ! Hello Maintainers , @Pavel401 , Kindly assign this Issue to me !

**Comment by Pavel401** [comment]
@hemant933  I have assigned you another issue you wanted to work on, #16.

@Tholkappiar, please make sure that the issue you want to work on is assigned to you. Don't start working on an issue unless you are assigned.

**Comment by Tholkappiar** [comment]
Yeah sure , thank you @Pavel401 !

**Comment by Pavel401** [comment]
Implemented in #32
